public_html/admin/model/sale/customer.php

```diff
@@ -1142,41 +1172,25 @@ public function sendApproveMail($customer_id)
         if ($customer_info && !$customer_info['approved']) {
             $this->load->language('mail/customer');
             $this->load->model('setting/store');
+            $languageId = $this->language->getContentLanguageID() ?? $this->language->getLanguageID();
             $store_info = $this->model_setting_store->getStore($customer_info['store_id']);
             if ($store_info) {
                 $store_info['store_url'] = $store_info['config_url'].'index.php?rt=account/login';
             } else {
                 $store_info = [
-                    'store_name'       => $this->config->get('store_name'),
-                    'store_url'        => $this->config->get('config_url').'index.php?rt=account/login',
-                    'config_mail_logo' => $this->config->get('config_mail_logo'),
-                    'config_logo'      => $this->config->get('config_logo'),
+                    'store_name'                    => $this->config->get('store_name'),
+                    'store_url'                     => $this->config->get('config_url').'index.php?rt=account/login',
+                    'config_mail_logo_'.$languageId => $this->config->get('config_mail_logo_'.$languageId),
```

Comment:
It is not a good idea to have dynamic key in the array. Not clear what to look for.